catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/QueryMetacardApplication.java

```diff
@@ -128,6 +131,7 @@ public void init() {
         (req, res) -> {
           String body = endpointUtil.safeGetBody(req);
           QueryBasic query = GSON.fromJson(body, QueryBasic.class);
+          query.setOwner(getSubjectEmail());
```

Comment:
@troymohl I think you have some experience in this area, do you know if we should use `getUniqueSubjectIdentifier`?
(https://github.com/codice/ddf/blob/b054ac205251e31390e5ccc9830dfe310a3f2659/platform/security/platform-security-core-api/src/main/java/ddf/security/SubjectIdentity.java#L27)

Reply:
getUniqueIdentifier is best as it would fallback to another value if the subject doesn't have an email address.